helpers/prompt.sh

```diff
@@ -71,3 +71,63 @@ PROMPT_BACKUP()
         echo "Backup created, size $(du -h "$PREFIX_BACKUP_DIR/$PREFIX_BACKUP_FILENAME" | cut -f1)"
     fi
 }
+
+PROMPT_DESKTOP_ENTRIES()
+{
+    CONTINUE=""
+    
+    if [[ "$(find "$HOME/.local/share/applications" -maxdepth 1 -iname 'ffxiv-run-act.desktop' -print -quit | wc -l)" -gt 0 ]]; then
+        echo "Desktop entries have already been created"
+        CONTINUE="n"
+    fi
+
+    while [[ "$CONTINUE" != "Y" && "$CONTINUE" != "N" && "$CONTINUE" != "y" && "$CONTINUE" != "n" ]]; do
+        read -p "Would you like to create desktop entries for FFXIV and ACT? [Y/N] " CONTINUE
+    done
+
+    if [[ "$CONTINUE" == "N" || "$CONTINUE" == "n" ]]; then
+        echo "Skipping entry creation"
+    else
+        mkdir -p $HOME/.local/share/applications &> /dev/null
+        if [[ "$(find "$HOME/.local/share/icons/hicolor/128x128/apps" -maxdepth 1 -iname 'lutris_final-fantasy-xiv-a-realm-reborn.png' -print -quit | wc -l)" -gt 0 ]]; then
+            FFXIV_ICON="Icon=$HOME/.local/share/icons/hicolor/128x128/apps/lutris_final-fantasy-xiv-a-realm-reborn.png"
+        elif [[ "$(find "$HOME/.local/share/icons/hicolor/32x32/apps" -maxdepth 1 -iname 'steam_icon_39210.png' -print -quit | wc -l)" -gt 0 ]]; then
+            FFXIV_ICON="Icon=$HOME/.local/share/icons/hicolor/32x32/apps/steam_icon_39210.png"
+        else
+            echo "Could not find FFXIV's icon. To manually correct this, add \"Icon=/path/to/desired/icon\" to \"$HOME/.local/share/applications/ffxiv-run-game.desktop\""
+	    FFXIV_ICON=""
+	fi
```

Comment:
Seems to be some indenting issues here (and throughout the file). Tabs vs spaces maybe? My indenting on these files is four spaces.
Also, the steam icon can be downloaded from here if it's not present:
<https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/39210/98527a5229540d86ced171d8a5dff2f8a560fe80.jpg>